### PR TITLE
[Merged by Bors] - fix: attempt to fix cpu consumption by reducing size of the output (PL-000) [bugfix]

### PR DESCRIPTION
**Pull request by zhihil**
**Fixes or implements VF-XXX**

### Related PRs

- https://github.com/voiceflow/XXXXXXXXX/pull/123

### Checklist

- [ ] Breaking changes have been communicated, including:
    - New required environment variables
    - Renaming of interfaces (API routes, request/response interface, etc)
- [ ] New environment variables have [been deployed](https://www.notion.so/voiceflow/Add-Environment-Variables-be1b0136479f45f1adece7995a7adbfb)
- [ ] Appropriate tests have been written
    - Bug fixes are accompanied by an updated or new test
    - New features are accompanied by a new test

### CODE REVIEW

**Review by xavidop — APPROVED** [review]

File: lib/clients/ai/openai/gpt3_5.ts [hunk 1]
```diff
@@ -42,7 +42,17 @@ export class GPT3_5 extends GPTAIModel {
...
       };
     } catch (error) {
-      log.warn(`GPT3.5 completion ${log.vars({ error: error?.response ?? error, messages, params })})}`);
+      const truncatedMessages = messages.slice(0, 10).map(({ content, ...rest }) => ({
+        ...rest,
+        content: content.substring(0, 200),
+      }));
+      log.warn(
+        `GPT3.5 completion ${log.vars({
+          error: error?.response ?? error,
```
Comment (trs) [inline lib/clients/ai/openai/gpt3_5.ts:51]: Can `error` just be `error.message`? And maybe add the code too?

```suggestion
          error: error?.message,
          code: error?.code,
```

**Review by trs — APPROVED** [review]

File: lib/clients/ai/openai/gpt3_5.ts (hunk 1 above)
Comment (zhihil) [reply lib/clients/ai/openai/gpt3_5.ts]: Sure

**Comment by zhihil** [comment]
bors cancel

**Comment by zhihil** [comment]
bors cancel

**Pull request closed without merging**